src/pulselistener/pulselistener/lib/pulse.py

```diff
@@ -123,6 +130,7 @@ def register(self, bus):
         body = json.loads(body.decode('utf-8'))
 
         # Push the message in the message bus
+        logger.debug('Received a pulse message')
```

Comment:
Nit: Maybe we should avoid logging this, as we get too many messages

Reply:
In debug it's never displayed by default, only when using a verbose mode while developing